src/main/java/com/slobodianyk/homework3/UserRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.slobodianyk.homework3;
+
+import com.slobodianyk.homework3.interfaces.UserRepository;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Optional;
+
+public class UserRepositoryJdbcImpl implements UserRepository {
+
+    // MySQL connection details
+    private static final String MYSQL_URL = "jdbc:mysql://localhost:3306/java_db";  // URL до вашої MySQL бази
+    private static final String MYSQL_USERNAME = "root";  // ваше MySQL ім'я користувача
```

Comment:
Chat GPT comments? 😄

Reply:
Yes, you are right, asked a couple of questions.